README.md

```diff
@@ -18,7 +18,7 @@ To get automated notifications of our latest release, you can watch the announce
 
 ## Quick Start
 
-    sudo docker run -d --restart=unless-stopped -p 80:80 -p 443:443 rancher/rancher
+    sudo docker run -d --restart=unless-stopped -p 80:80 -p 443:443 --privileged rancher/rancher
```

Comment:
Can we add a quick explanation why `--privileged` is needed?

Reply:
Sure, I'd prefer to put in the docs page we link below though, or lower in the readme. The quick start is just 2 lines and I feel adding it here removes some of the wow factor. I have to do a separate docs PR anyway.